pkg/gardenlet/controller/shoot/care/reconciler.go

```diff
@@ -107,9 +107,15 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		gardencorev1beta1.ShootAPIServerAvailable,
 		gardencorev1beta1.ShootControlPlaneHealthy,
```

Comment:
will the list of components whose state collectively define the health of `ShootControlPlaneHealthy` be different for workerless shoot and shoot with workers? If yes where is this listed defined?